aiven/client/cli.py

```diff
@@ -255,6 +275,21 @@ def enter_password(self, prompt, var="AIVEN_PASSWORD", confirm=False):
 
         return password
 
+    def print_boxed(self, lines: List[str]) -> None:
+        longest = max(len(line) for line in lines)
+
+        print("*" * longest)
+        for line in lines:
+            print(line)
+        print("*" * longest)
+
+    def confirm(self, prompt: str = "confirm? "):
```

Comment:
This could have some hint also (e.g. `.. (y/N)? ` as default seems to be effectively N).